<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionInformationDialog</class>
 <widget class="QDialog" name="ConnectionInformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>379</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>L2TP IPSec VPN</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="m_pDialogLayout">
   <item>
    <widget class="QTabWidget" name="m_pTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_pInfoTab">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="m_pGatewayLabel">
         <property name="text">
          <string>Remote Server:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="m_pGateway">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="m_pInterfaceNameLabel">
         <property name="text">
          <string>PtP Interface:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="m_pInterfaceName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="m_pRemoteHostLabel">
         <property name="text">
          <string>Destination Address:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="m_pRemoteHost">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="m_pIPlAddressLabel">
         <property name="text">
          <string>IP Address:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="m_pIPAddress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="m_pSubnetMaskLabel">
         <property name="text">
          <string>Subnet Mask:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="m_pSubnetMask">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="m_pPrimaryDNSLabel">
         <property name="text">
          <string>Primary DNS:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="m_pPrimaryDNS">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="m_pSecondaryDNSLabel">
         <property name="text">
          <string>Secondary DNS:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="m_pSecondaryDNS">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="m_pTenaryDNSLabel">
         <property name="text">
          <string>Tenary DNS:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="m_pTenaryDNS">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QPushButton" name="m_pStatisticsPushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>85</horstretch>
           <verstretch>27</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Statistics ...</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_pInternetInterfaceNameLabel">
         <property name="text">
          <string>Internet Interface:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="m_pInternetInterfaceName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_pInternetInterfaceGatewayLabel">
         <property name="text">
          <string>Internet Interface Gateway:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="m_pInternetInterfaceGateway">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pLogTab">
      <attribute name="title">
       <string>Logs</string>
      </attribute>
      <layout class="QVBoxLayout" name="m_pLogTabLayout">
       <item>
        <widget class="QTextEdit" name="m_pTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_pButtonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pTabWidget</tabstop>
  <tabstop>m_pStatisticsPushButton</tabstop>
  <tabstop>m_pButtonBox</tabstop>
  <tabstop>m_pTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/L2tpIPsecVpn.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ConnectionInformationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
